content/blog/biscuit-2.0.md

```diff
@@ -0,0 +1,128 @@
++++
+title = "Biscuit 2.0 release"
+description = "Introducing Biscuit 2.0"
+date = 2022-01-16T09:19:42+00:00
+updated = 2022-14-01T09:19:42+00:00
+draft = true
+template = "blog/page.html"
+
+[taxonomies]
+authors = ["Rustaceans"]
+
+[extra]
+lead = "Biscuit 2.0 announcement"
++++
+
+We are delighted to announce the release of Biscuit at version 2.0! Biscuit is a specification
+for a cryptographically verified authorization token supporting offline delegation, and a
+language for authorization policies based on Datalog. It has been in development for 3 years
+and is already deployed in production systems.
+
+Biscuit was designed to address new challenges in building authorization in distributed systems:
+- a query going through microservices must be verifiable decentrally without giving its full
+authrization level to every node
+- API authorization increasingly has to fit a user's authorization landscape (teams, roles,
+delegation) while guaranteeing multitenant isolation
+
+As an example, [Clever Cloud](https://www.clever-cloud.com)'s Pulsar service is accessible
+with Biscuit tokens: each customer is provided a token with full access to their namespace,
```

Comment:
```suggestion
with Biscuit tokens: each customer is given a token with full access to their namespace,
```

(else it would be "provided with", which would be a bit redundant)